src/Database/Connection.php

```diff
@@ -158,10 +159,11 @@ public function driver($driver = null, $config = [])
             return $this->_driver;
         }
         if (is_string($driver)) {
-            if (!class_exists($driver)) {
+            $className = App::className($driver, 'Database/Driver');
+            if (!$className || ($className && !class_exists($className))) {
```

Comment:
Silly me, I had something totally different in mind, trying to avoid "unnecessary" `class_exists()` calls, which, now that I think about it again, actually isn't possible without additional checks...